third-party/README.md

```diff
@@ -8,6 +8,7 @@ Interested in writing an extension? It's easy! Take a look at ["Writing Your Own
 
 - OSGi Bundle Packaging Plugin Extension ([Maven](https://github.com/thought-gang/jib-maven-plugin-extension.git)): an extension to containerize an OSGI bundle (Maven packaging type `bundle`)
 - Layer With Modification Time ([Maven](https://github.com/infobip/jib-layer-with-modification-time-extension-maven)): an extension for selectively setting file timestamps to build time (eg. for hosted web resources)
+- Javaagent Attachment Plugin Extension ([Gradle](https://github.com/ryandens/javaagent-gradle-plugin#jib-integration)): An extension that allows you to automatically add a javaagent from a Maven repository as a layer in your OCI image built by jib and modifies the entrypoint of the image to include the `-javaagent` flag.
```

Comment:
I see this isn't specific to OCI images (doesn't matter if the format is OCI or Docker), so maybe this:
```suggestion
- Javaagent Attachment Plugin Extension ([Gradle](https://github.com/ryandens/javaagent-gradle-plugin#jib-integration)): An extension that allows you to automatically add a javaagent from a Maven repository as a layer in your container image built by Jib and modifies the entrypoint of the image to include the `-javaagent` flag.
```